### DISCUSSION TITLE
New Page Idea: Long Term Promises

### DISCUSSION THREAD

**Discussion by Maraza25** [octoposprime/op-mo-app#7] [Ideas]
How do you think it would be if you add a page in our mobile application where we talk about what we promise to users in the future?

@octoposprime/mo

**Comment by doganabdussamed** [comment]
I think it can be beautiful, but I don't know the effect.

**Comment by aminenurgynk** [comment]
It sounds good

**Comment by Maraza25** [comment]
How do you want the page design to be?

**Comment by Maraza25** [comment]
I believe it makes sense to use square cards in design, with an icon on top and a title below each card. It would also be appropriate to present the details in a dialog that opens when the cards are clicked
